x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccountsTests.java

```diff
@@ -372,6 +372,7 @@ public void testElasticEnterpriseSearchServerAccount() {
         });
 
         final IndexAbstraction elasticsearchIndex = mockIndexAbstraction("search-" + randomAlphaOfLengthBetween(1, 20));
+        assertThat(role.indices().allowedIndicesMatcher("indices:foo").test(elasticsearchIndex), is(false));
```

Comment:
Privilege `all` allows any action begining with `indices:`. So this assertion is incorrect.

Reply:
Thanks @ywangd , updated the list of index privileges.